### ISSUE TITLE
Transformer output

### ISSUE THREAD

**Issue by narley** [tim-crisp/javascript.lol#6]
Is there a way to display the output of each transformer in its own block? Like a "Show result" button.

**Comment by tim-crisp** [comment]
Good idea. As an interim solution it might be easier to log each transformer output in the console as the console is relatively redundant in transformer mode at the moment.
